tools/aot/execute_test.template.cpp

```diff
@@ -100,35 +100,35 @@ TEST(AotCompiled, ExecuteTest) {
       // "test/AotCompile/_internal_add_mul_single_output_reference_tensors.npz"
   );
 
-  //##READ_REFERENCE_TENSORS_INTO_NPY_ARRAY##//
-  // cnpy::NpyArray refInput0 = reference_tensors["Input0"];
-  // cnpy::NpyArray refInput1 = reference_tensors["Input1"];
-  // cnpy::NpyArray refInput2 = reference_tensors["Input2"];
-  // cnpy::NpyArray refOutput0 = reference_tensors["Output0"];
+  // ##READ_REFERENCE_TENSORS_INTO_NPY_ARRAY##//
+  //  cnpy::NpyArray refInput0 = reference_tensors["Input0"];
+  //  cnpy::NpyArray refInput1 = reference_tensors["Input1"];
+  //  cnpy::NpyArray refInput2 = reference_tensors["Input2"];
+  //  cnpy::NpyArray refOutput0 = reference_tensors["Output0"];
```

Comment:
I'd avoid clang-format changes on this file, since it might break some of the match-replace patterns when we instantiate this template in `aot_compile`. You might want to ignore this file from clang-format, or make sure the patterns are correctly updated.